funcx_endpoint/funcx_endpoint/cli.py

```diff
@@ -169,6 +171,28 @@ def start_endpoint(*, name: str, endpoint_uuid: str | None):
     )
 
 
+@app.command(name="logout", help="Logout from all endpoints")
+@click.option("--environment", default=None, help="The environment to logout from")
+@common_options
+def logout_endpoint(environment: str | None):
+    """
+    Logout from all endpoints and remove cached authentication credentials
+
+    TODO Currently there does not seem to be credentials associated with
+     specific endpoints, so will just have to remove all tokens for
+     all environments using the default LoginManager.
+
+     If environment is specified, it will only clear credentials for
+     that environment (namespace in token_storage cached tokens)
+     Alternatively, environment can be retrieved from the endpoint
+     config.py if endpoint is specified, but that is a much more
+     convoluted way to get the environment
+    """
+    tokens_revoked = LoginManager(environment=environment).logout()
```

Comment:
One thing it would be good to do before logging out is checking if there are any running endpoints locally.  If there are, it wouldn't be nice to yank their tokens from underneath them since they will end up dying in unpredictable ways.  It would probably be better to ask the user to stop any running endpoints, with possibly a `--force` just do it anyway.

Reply:
Will add checking for running endpoints and a --force option.

Reply:
Added check for running and the --force option